Combobox.js

```diff
@@ -767,6 +875,22 @@ define([
 			};
 		}),
 
+		// HasDropDown#_dropDownKeyUpHandler() override.
+		// Do not call openDropDown if widget does not have a down arrow shown.
+		_dropDownKeyUpHandler: dcl.superCall(function (sup) {
+			return function () {
+				if (this.hasDownArrow) {
+					if (this.inputNode.value.length === this.minFilterChars === 0) {
+						this.list.query = this.getInitialQuery;
```

Comment:
This must be wrong because you aren't accounting for `getInitialQuery` being a function.  In other parts of the code you do account for that.